### PR TITLE
madspin for powheg+madpsin+pythia8 single top 4f tch, additionally for TDR 14TEV samples with the same settings as Moriond17 production

### PR DESCRIPTION
**Pull request by Andrej-CMS**
Please check the configs!

Regarding the run_card.dat settings: should the reweight_PDF setting be set to $DEFAULT_PDF_MEMBERS?

If I'm not mistaken these variations are already calculated by Powheg.

### CODE REVIEW

**Comment by Andrej-CMS** [comment]
Please check also the ttH settings for 14TeV TDR samples

**Comment by Andrej-CMS** [comment]
the TDR 14 TeV samples have the same settings as the 13TeV samples, just with a different center of mass energy

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/st_tch_4f_ckm_NLO_top_powheg_madspin/st_tch_4f_ckm_NLO_top_powheg_madspin_madspin_card.dat [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#************************************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/st_tch_4f_ckm_NLO_top_powheg_madspin/st_tch_4f_ckm_NLO_top_powheg_madspin_madspin_card.dat]: this file is called
`st_tch_4f_ckm_NLO_top_powheg_madspin_madspin_card.dat`
and should probably be called
`st_tch_4f_ckm_NLO_top_powheg_madspin_card.dat`
right?

File: bin/Powheg/production/pre2017/14TeV/ttH_inclusive_hdamp_NNPDF30_14TeV_M125/ttH_inclusive_hdamp_NNPDF30_14TeV_M125.input [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+tdec/mumass 0.1057
+tdec/taumass 1.777
+tdec/dmass   0.100
+tdec/umass   0.100
+tdec/smass   0.200
+tdec/cmass   1.5
+tdec/sin2cabibbo 0.051
+
+# parameters for parallel runs (remove '#' below for parallel runs):
+#manyseeds  1       ! Used to perform multiple runs with different random
```
Comment (perrozzi) [inline bin/Powheg/production/pre2017/14TeV/ttH_inclusive_hdamp_NNPDF30_14TeV_M125/ttH_inclusive_hdamp_NNPDF30_14TeV_M125.input]: remove commented parameters

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll_customizecards.dat [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+#set param_card mass 6 173.0
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll_customizecards.dat]: remove commented parameters

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/st_tch_4f_ckm_NLO_top_powheg_madspin/st_tch_4f_ckm_NLO_top_powheg_madspin_card.dat [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+# set BW_cut 15                # cut on how far the particle can be off-shell
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/st_tch_4f_ckm_NLO_top_powheg_madspin/st_tch_4f_ckm_NLO_top_powheg_madspin_card.dat]: I still see commented lines...

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll_madspin_card.dat [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+ #set BW_cut 15                # cut on how far the particle can be off-shell
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll_madspin_card.dat]: also this...

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll_proc_card.dat [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import model loop_sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model loop_sm-no_b_mass
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll/vh012j_5f_NLO_FXFX_M125_14TeV_VToAll_proc_card.dat]: please remove the commented command

File: bin/Powheg/production/pre2017/14TeV/ttH_inclusive_hdamp_NNPDF30_14TeV_M125/ttH_inclusive_hdamp_NNPDF30_14TeV_M125.input [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+! Higgs decay:
+hdecaymode -1   ! -1 no decay
+                ! 0 all decay channels open
+                ! 1-6 d dbar, u ubar,..., t tbar
+                ! 7-9 e+ e-, mu+ mu-, tau+ tau-
+                ! 10  W+W-
+                ! 11  ZZ
+                ! 12  gamma gamma
+
+#pythiatune 320 ! PYTHIA tune
```
Comment (efeyazgan) [inline bin/Powheg/production/pre2017/14TeV/ttH_inclusive_hdamp_NNPDF30_14TeV_M125/ttH_inclusive_hdamp_NNPDF30_14TeV_M125.input]: commented parameter

File: bin/Powheg/production/pre2017/14TeV/ttH_inclusive_hdamp_NNPDF30_14TeV_M125/ttH_inclusive_hdamp_NNPDF30_14TeV_M125.input [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+ih1   1           ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2   1           ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 7000d0     ! energy of beam 1
+ebeam2 7000d0     ! energy of beam 2
+
+# https://lhapdf.hepforge.org/pdfsets.html
+lhans1  260000      ! pdf set for hadron 1 (LHA numbering)
+lhans2  260000      ! pdf set for hadron 2 (LHA numbering)
+
+#bornsuppfact 1d0 ! (default 0d0) mass param for Born suppression factor.
```
Comment (efeyazgan) [inline bin/Powheg/production/pre2017/14TeV/ttH_inclusive_hdamp_NNPDF30_14TeV_M125/ttH_inclusive_hdamp_NNPDF30_14TeV_M125.input]: commented parameter

**Comment by efeyazgan** [comment]
there are still some commented parameters and comments

**Comment by perrozzi** [comment]
why there are vh and tth datacards in the pr too?!?

**Comment by Andrej-CMS** [comment]
They were added because TDR wants their "highest priority" samples to be processes as fast as possible. Do you want me to make a separate PR for this?

**Comment by perrozzi** [comment]
things look ok to me now

**Pull request merged**